paper-tab.html

```diff
@@ -76,7 +76,7 @@
     }
 
     #ink {
-      color: #ffff8d;
+      color: var(--paper-tab-ink, #ffff8d);
```

Comment:
Nice usage of default value :+1: 
We should probably use the color palette from paper-styles' color.html
e.g.
```
color: var(--paper-tab-ink, --paper-yellow-a100);
```